src/main/java/com/tistory/povia/realworld/common/controller/ExceptionHandlingController.java

```diff
@@ -0,0 +1,27 @@
+package com.tistory.povia.realworld.common.controller;
+
+import com.tistory.povia.realworld.user.exception.DuplicatedEmailException;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ExceptionHandlingController {
+
+    private final Logger log = LoggerFactory.getLogger(getClass());
+
+    @ExceptionHandler({IllegalArgumentException.class, DuplicatedEmailException.class})
+    public ResponseEntity<?> handleBadRequestException(Exception exception) {
```

Comment:
```suggestion
    private ResponseEntity<?> handleBadRequestException(Exception exception) {
```
`@RestControlelrAdvice`는 public하게 메소드를 호출하지 않는데 private이나 protected로 방어하는 것이 좋지 않을까요?

Reply:
Spring AOP 포인트 컷 기반으로 돌아거는 걸로 아는데   
프록시가 있으니 protected가 낫겠군요